xbmc/pvr/dialogs/GUIDialogPVRChannelsOSD.cpp

```diff
@@ -183,14 +181,11 @@ CPVRChannelGroupPtr CGUIDialogPVRChannelsOSD::GetPlayingGroup()
 
 void CGUIDialogPVRChannelsOSD::Update()
 {
+  g_infoManager.RegisterObserver(this);
```

Comment:
@FernetMenta is this okay, Thought there's no reason to hold the graphicsContext while grabbing locks in g_infoManager and g_EpgContainer.